### PR TITLE
Added custom tooltip implementation for dashboard to show info for each metric

### LINKED ISSUE #269: Add an info icon beside each metric in the dashboard

### PR DESCRIPTION
**Pull request by dalmia**
Fixes #269 

## Summary
- adds an i icon beside each metric - upon hovering (on laptop) or touching (on mobile) - shows info about the metric

## Test Plan
Tested locally.

### CODE REVIEW

File: src/assets/images/info.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg height='100px' width='100px'  fill="#000000" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 450 450" x="0px" y="0px"><title>Fichier 2locate</title><g data-name="Calque 2"><g data-name="Calque 2"><circle cx="225" cy="133.5" r="31.25"></circle><path d="M251,219.12v103a26,26,0,0,1-52,0v-103a26,26,0,0,1,52,0Z"></path><path d="M225,0C100.74,0,0,100.74,0,225S100.74,450,225,450,450,349.26,450,225,349.26,0,225,0ZM355.81,355.81A185,185,0,1,1,94.19,94.19,185,185,0,1,1,355.81,355.81Z"></path><path d="M225,0C100.74,0,0,100.74,0,225S100.74,450,225,450,450,349.26,450,225,349.26,0,225,0ZM355.81,355.81A185,185,0,1,1,94.19,94.19,185,185,0,1,1,355.81,355.81Z"></path></g></g></svg>
```
Comment (deepansh96) [inline src/assets/images/info.svg:1]: There's an already existing svg called `exclamation-circle-solid` which was being used in that message "only indian numbers allowed"
either use that here or use this one there, no need for two different svgs

File: src/pages/Dashboard.vue [hunk 2]
```diff
@@ -161,12 +239,38 @@
...
                 </p>
-                <p :class="cardMetricTitleClass">
-                  {{ $t("dashboard.summary.num_questions_answered") }}
-                </p>
+                <div :class="cardMetricTitleClass" class="flex justify-center">
+                  <div
+                    class="flex flex-col md:flex-row space-y-2 md:space-y-0 md:space-x-2 items-center px-1"
+                  >
+                    <!-- title -->
+                    <p>
```
Comment (deepansh96) [inline src/pages/Dashboard.vue:251]:
```suggestion
                    <p class="leading-4">
```
Without this the metric title is not centered with respect to the  `i` icon

File: src/pages/Dashboard.vue [hunk 3]
```diff
@@ -105,12 +105,38 @@
...
               <p :class="cardMetricValueClass">
                 {{ completionRate }}
               </p>
-              <p :class="cardMetricTitleClass">
-                {{ $t("dashboard.summary.completion_rate") }}
-              </p>
+              <div :class="cardMetricTitleClass" class="flex justify-center">
+                <div
+                  class="flex flex-col md:flex-row space-y-2 md:space-y-0 md:space-x-2 items-center"
+                >
```
Comment (deepansh96) [inline src/pages/Dashboard.vue:115]: Can combine these two divs right?

File: src/pages/Dashboard.vue [hunk 4]
```diff
@@ -105,12 +105,38 @@
...
+                    {{ $t("dashboard.summary.completion_rate.title") }}
+                  </p>
+                  <div class="flex relative">
+                    <!-- info icon -->
+                    <inline-svg
+                      :src="require('@/assets/images/info.svg')"
+                      class="h-4 w-4 text-yellow-900 fill-current hover:cursor-pointer"
+                      @mouseover="showCompletedHelpText = true"
+                      @mouseout="showCompletedHelpText = false"
+                      @touchmove="showCompletedHelpText = true"
```
Comment (deepansh96) [inline src/pages/Dashboard.vue:127]: why not touchstart?

File: src/pages/Dashboard.vue [hunk 5]
```diff
@@ -105,12 +105,38 @@
...
+                      @touchend="showCompletedHelpText = false"
+                    ></inline-svg>
+                    <!-- info text -->
+                    <p
+                      class="bg-black text-white rounded-md p-2 absolute ml-4 w-28"
```
Comment (deepansh96) [inline src/pages/Dashboard.vue:132]: Screenshot 2021-05-19 at 7 46 47 PM
Some css issue?

Comment (deepansh96) [reply src/pages/Dashboard.vue:132]: Screenshot 2021-05-19 at 7 47 31 PM
In 320px mode, the info text is being cut off on the right side

**Review by deepansh96 — CHANGES_REQUESTED** [review]

File: src/assets/locales/en.js [hunk 6]
```diff
@@ -285,10 +285,24 @@ export default {
...
-      accuracy: "ACCURACY",
-      num_questions_answered: "QUESTIONS ANSWERED",
+      completion_rate: {
+        title: "COMPLETED",
+        tooltip: "Percentage of viewers who answered all the questions",
+      },
+      one_minute_retention: {
+        title: "RETENTION AT 1 MINUTE",
+        tooltip:
+          "Percentage of viewers who watched your video for at least 1 minute",
```
[the remark this answers is not in the document]
Comment (dalmia) [reply src/assets/locales/en.js:295]: beyond 1 minute

**Review by deepansh96 — APPROVED** [review]

**Pull request merged**